libnmstate/nm/ipv4.py

```diff
@@ -31,6 +31,10 @@ def create_setting(config, base_con_profile):
             setting_ipv4.props.ignore_auto_routes = False
             setting_ipv4.props.never_default = False
             setting_ipv4.props.ignore_auto_dns = False
+            setting_ipv4.props.gateway = None
```

Comment:
As a future refactoring step it would be great to make the ip settings objects and have the common code in a base class that ipv4/ipv6 can then inherit from.

Reply:
Thanks. I will consider it in another PR.